packages/frontend/src/components/header/ActivityHeader.tsx

```diff
@@ -20,8 +23,25 @@ export function ActivityHeader(props: ActivityHeaderProps) {
         <p className="text-gray-500 dark:text-gray-600 hidden md:block">
           Transactions per second
         </p>
-        <p className="text-gray-500 dark:text-gray-600 text-xs md:text-base text-right w-full md:w-auto">
-          More txs on Ethereum because of L2s
+
+        <p
+          className={cx(
+            'text-gray-500 dark:text-gray-600 text-xs md:text-base text-right w-full md:w-auto',
+            'flex items-center gap-1.5',
+          )}
+        >
+          Observed over the last 7 days
+          <span
+            className="Tooltip inline-block"
+            title="More txs on Ethereum because of L2s. Formula: (L2 txs + ETH txs) / ETH txs"
```

Comment:
```suggestion
            title="How many more transactions are settled by Ethereum if we take into account the L2 solutions. Formula: (L2 txs/7d + ETH txs/7d) / ETH txs/7d"
```

Reply:
Changed the tooltip so it looks like this: 
<img width="429" alt="Screenshot 2022-10-18 at 09 06 16" src="https://user-images.githubusercontent.com/38423054/196360565-f3c66183-d31f-49dc-8099-8b6ba5168faf.png">